code/lib/preview-api/src/modules/preview-web/docs-context/DocsContextProps.ts

```diff
@@ -0,0 +1,3 @@
+import type { DocsContextProps } from '@storybook/types';
+
+export { DocsContextProps };
```

Comment:
For curiosity. Is a re-export for types possible?

```suggestion
export type { DocsContextProps } from '@storybook/types';
```